addons/account/models/res_config_settings.py

```diff
@@ -94,9 +94,6 @@ class ResConfigSettings(models.TransientModel):
     module_account_iso20022 = fields.Boolean(string='SEPA Credit Transfer / ISO20022')
     module_account_sepa_direct_debit = fields.Boolean(string='Use SEPA Direct Debit')
     module_account_bank_statement_import_qif = fields.Boolean("Import .qif files")
-    module_account_bank_statement_import_ofx = fields.Boolean("Import in .ofx format")
-    module_account_bank_statement_import_csv = fields.Boolean("Import in .csv, .xls, and .xlsx format")
-    module_account_bank_statement_import_camt = fields.Boolean("Import in CAMT.053 format")
```

Comment:
you can't do that. Do you understand that you're modifying the settings for everyone, not only for the indian companies?? :facepalm:

Reply:
@qdp-odoo this is based on this [discussion](https://discord.com/channels/678381219515465750/687337689988726949/1314599750821216358)

this module is auto-install so the settings do not make sense so we are thinking of removing it.
but if we miss something then plz suggest.

Reply:
ok, I didn't know there were auto_installed :-)